tuf/store/filestore.go

```diff
@@ -43,6 +43,10 @@ type FilesystemStore struct {
 func (f *FilesystemStore) GetMeta(name string, size int64) ([]byte, error) {
 	fileName := fmt.Sprintf("%s.%s", name, f.metaExtension)
 	path := filepath.Join(f.metaDir, fileName)
+	if _, err := os.Stat(path); os.IsNotExist(err) {
```

Comment:
Oh nice!  For some reason I thought that only worked on the result of `os.Stat`, thanks!